### PR TITLE
Update issue-260.markdown

### PR DESCRIPTION
**Pull request by friedbrice**
Submitting a blog post for your consideration.

### CODE REVIEW

File: data/newsletter/issue-260.markdown [hunk 1]
```diff
@@ -4,7 +4,8 @@ This is a weekly summary of what's going on in its community.
 
 ## Featured
 
-undefined
+- [Of function instances and abstract syntax](https://www.danielbrice.net/blog/of-function-instances-and-abstract-syntax/)
```
Comment (tfausak) [inline data/newsletter/issue-260.markdown:7]: Would you like attribution? 
```suggestion
- [Of function instances and abstract syntax](https://www.danielbrice.net/blog/of-function-instances-and-abstract-syntax/) by Daniel Brice
```

**Review by tfausak — APPROVED** [review]
Thanks for the contribution. Looks great!

File: data/newsletter/issue-260.markdown (hunk 1 above)
Comment (friedbrice) [reply data/newsletter/issue-260.markdown:7]: Yes, please. It just didn't occur to me earlier.

**Comment by friedbrice** [comment]
Thanks for the opportunity to share :-)

**Pull request merged**